module3/projects/take_home/rubric.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Take-Home Rubric
+layout: page
+---
+
+In order for a candidate to pass this challenge, **_they must score "Leaning Yes Hire" or "Strong Yes Hire" in every category._**
+
+<section class="dropdown">
+### 1. Problem Solving and Task Completion
+#### Strong Yes Hire
+
+The candidate...
+- is able to articulate multiple approaches and describe the pros and cons of each and is able to fully implement the approach, without any real help from the interviewer.
+- recognizes edge cases, draws attention to them unprompted, and solves them.
+- goes above and beyond the MVP in either adding new functionality beyond the base requirements or special consideration through means of testing or accessibility. 
+
+#### Leaning Yes Hire
+The candidate...
+- is able to outline a solution beyond the most basic solution, covering the vast majority of edge cases. 
+- asks questions to clarify the scope of the problem or states their assumptions without being prompted. 
+- is able to modify their solution independently if an edge case is pointed out.
+- builds a solution that meets all MVP expectations provided by the challenge and is easy to use by the end user.
+
+#### Leaning Don’t Hire
+The candidate...
+- is unable to arrive at a solution beyond the most basic approach. The final solution does not account for edge cases. 
+- dives in to a solution without giving the problem much thought. 
+- is only able to solve a bug or for an identified issue with significant help.
+- built most of the MVP, but one or two requirements may be incomplete.
+
+#### Strong Don’t Hire
+The candidate...
+- is unable to arrive at a solution beyond the most basic approach. 
+- fails to complete three or more requirements of the MVP or the application is broken.
+</section>
+
+<section class="dropdown">
+### 2. Technical Fluency and Code Design
+#### Strong Yes Hire
+
+The candidate...
+- chooses to use standard library functions and is able to describe their behavior when asked. 
+- is able to break down a complex system into elegantly structured components, and thoroughly describe the interaction model, the interface and the behavior Abstractions are clear and clean, given the problem definition, and keep open the possibility of future needs without significant refactoring.
+- understands time and space complexity (even if they don’t use those terms) and can describe the characteristics of their implemented code if appropriate.
+
+#### Leaning Yes Hire
+The candidate...
+- can describe the tradeoffs inherent in different approaches, and can articulate why a specific data structure is appropriate. 
+- chooses to use standard library functions and is able to describe their behavior when asked (ex: chooses ActiveRecord over plain Ruby). 
+- is able to structure their solution in a way that separates classes appropriately and demonstrates a good understanding of OOP principles (even if they don’t use the specific terms). 
+
```

Comment:
-  makes intentional decisions about their React component structure and utilizes state and props to manage data efficiently 
- correctly uses `useEffect` to handle the asynchronous behavior of network requests